ctapipe/io/containers.py

```diff
@@ -285,7 +285,33 @@ class MCHeaderContainer(Container):
     prod_site_coord = Field("None", "site (long., lat.) coordinates")
     prod_site_subarray = Field("None", "site subarray")
     spectral_index = Field(0.0, "Power-law spectral index of spectrum")
-
+    shower_prog_start = Field(0.0, """Time when shower simulation started,
+                              CORSIKA: only date""")
+    shower_prog_id = Field(0.0, "CORSIKA=1, ALTAI=2, KASCADE=3, MOCCA=4")
+    detector_prog_start = Field(0.0, "Time when detector simulation started")
+    detector_prog_id = Field(0.0, "simtelarray=1")
+    num_showers = Field(0.0, "Number of showers simulated")
+    shower_reuse = Field(0.0, "Numbers of uses of each shower")
+    max_alt = Field(0.0, "Maximimum shower altitude", unit=u.rad)
+    min_alt = Field(0.0, "Minimum shower altitude", unit=u.rad)
+    max_az = Field(0.0, "Maximum shower azimuth", unit=u.rad)
+    min_az = Field(0.0, "Minimum shower azimuth", unit=u.rad)
+    diffuse = Field(0.0, "Diffuse Mode On/Off")
+    max_viewcone_radius = Field(0.0, "Maximum viewcone radius", unit=u.deg)
+    min_viewcone_radius = Field(0.0, "Minimum viewcone radius", unit=u.deg)
+    max_scatter_range = Field(0.0, "Maximum scatter range", unit=u.m)
+    min_scatter_range = Field(0.0, "Minimum scatter range", unit=u.m)
+    core_pos_mode = Field(0.0, "Core Position Mode (fixed/circular/...)")
+    injection_height = Field(0.0, "Height of particle injection", unit=u.m)
+    atmosphere = Field(0.0, "Atmospheric model number")
+    corsika_iact_options = Field(0.0, "Detector MC information")
+    corsika_low_E_model = Field(0.0, "Detector MC information")
+    corsika_high_E_model = Field(0.0, "Detector MC information")
+    corsika_bunchsize = Field(0.0, "Detector MC information")
+    corsika_wlen_min = Field(0.0, "Detector MC information")
```

Comment:
shouldn't a unit be there as well?

Reply:
>I think we should add the run id (or obsid), however we want to call it, to this container.

That can be in a separate container that is written at the same time as this one - otherwise we'd have to reproduce it for every container we want to have a run_id, etc in.  However, having the corsika run id might be useful, since so far the "obs_id" is filled as the simtelarray run id, but we lose the original corsika id (which can be different).